Any idea why LDC 32-bit on Travis is failing?

```
Test failed.  The logged output:
../generated/linux/release/32/dmd -conf= -m32 -Irunnable -g  -odtest_results/runnable -oftest_results/runnable/gdb10311_0 runnable/gdb10311.d
gdb test_results/runnable/gdb10311_0 --batch -x test_results/runnable/gdb10311_0.gdb
Breakpoint 1 at 0x8048632: file runnable/gdb10311.d, line 19.
test_results/runnable/gdb10311_0.gdb:3: Error in sourced command file:
During startup program exited normally.
==============================
Test failed: 
GDB regex: 'RESULT=.*33' didn't match output:
----
Breakpoint 1 at 0x8048632: file runnable/gdb10311.d, line 19.
test_results/runnable/gdb10311_0.gdb:3: Error in sourced command file:
During startup program exited normally.
----
make[1]: *** [test_results/runnable/gdb10311.d.out] Error 1
make: *** [start_runnable_tests] Error 2
make: *** Waiting for unfinished jobs....
```

This looks pretty unrelated as it Semaphore with LDC 64-bit does pass.

--- a/docs/Makefile
+++ b/docs/Makefile
@@ -22,7 +22,7 @@ export DIFFABLE
 
 build: $(addprefix $G/, $(DMD_MAN_PAGE) $(OTHER_MAN_PAGES))
 
-preview:
+preview: $(MAN_PAGE)
 	man -l $G/$(DMD_MAN_PAGE)
 
 ################################################################################

--- a/src/dmd/cli.d
+++ b/src/dmd/cli.d
@@ -67,14 +67,6 @@ bool isCurrentTargetOS(TargetOS os)
     return (os & targetOS) > 0;
 }
 
-/// Representation of a CLI `Option`
-struct Option
-{
-    string flag; /// The CLI flag without leading `-`, e.g. `color`
-    string helpText; /// A detailed description of the flag
-    TargetOS os = TargetOS.all; /// For which `TargetOS` the flags are applicable
-}
-
 /**
 Contains all available CLI $(LREF Option)s.
 
@@ -82,6 +74,14 @@ See_Also: $(LREF Option)
 */
 struct Usage
 {
+    /// Representation of a CLI `Option`
+    struct Option
+    {
+        string flag; /// The CLI flag without leading `-`, e.g. `color`
+        string helpText; /// A detailed description of the flag
+        TargetOS os = TargetOS.all; /// For which `TargetOS` the flags are applicable
+    }
+
     /// Returns all available CLI options
     static immutable options = [
         Option("allinst",
@@ -316,6 +316,31 @@ struct Usage
             "write JSON file to filename"
         ),
     ];
+
+    /// Representation of a CLI transition
+    struct Transition
+    {
+        string bugzillaNumber; /// bugzilla issue number (if existent)
+        string name; /// name of the transition
+        string paramName; // internal transition parameter name
+        string helpText; // detailed description of the transition
+    }
+
+    /// Returns all available transitions
+    static immutable transitions = [
+        Transition("3449", "field", "vfield",
+            "list all non-mutable fields which occupy an object instance"),
+        Transition("10378", "import", "bug10378",
+            "revert to single phase name lookup"),
+        Transition(null, "checkimports", "check10378",
+            "give deprecation messages about 10378 anomalies"),
+        Transition("14488", "complex", "vcomplex",
+            "give deprecation messages about all usages of complex or imaginary types"),
+        Transition("16997", "intpromote", "fix16997",
+            "fix integral promotions for unary + - ~ operators"),
+        Transition(null, "tls", "vtls",
+            "list all variables going into thread local storage"),
+    ];
 }
 
 /**
@@ -377,14 +402,29 @@ CPU architectures supported by -mcpu=id:
     /// Language changes listed by -transition=id
     static string transitionUsage()
     {
-        return "Language changes listed by -transition=id:
-  =all           list information on all language changes
-  =checkimports  give deprecation messages about 10378 anomalies
-  =complex,14488 give deprecation messages about all usages of complex or imaginary types
-  =field,3449    list all non-mutable fields which occupy an object instance
-  =import,10378  revert to single phase name lookup
-  =intpromote,16997 fix integral promotions for unary + - ~ operators
-  =tls           list all variables going into thread local storage
+        enum maxFlagLength = 20;
+        enum s = () {
+            auto buf = "Language changes listed by -transition=id:
 ";
+            auto allTransitions = [Usage.Transition(null, "all", null,
+                "list information on all language changes")] ~ Usage.transitions;
+            foreach (t; allTransitions)
+            {
+                buf ~= "  =";
+                buf ~= t.name;
+                auto lineLength = 3 + t.name.length;
+                if (t.bugzillaNumber !is null)
+                {
+                    buf ~= "," ~ t.bugzillaNumber;
+                    lineLength += t.bugzillaNumber.length + 1;
+                }
+                foreach (i; 0 .. maxFlagLength - lineLength)
+                    buf ~= " ";
+                buf ~= t.helpText;
+                buf ~= "\n";
+            }
+            return buf;
+        }();
+        return s;
     }
 }

--- a/src/dmd/mars.d
+++ b/src/dmd/mars.d
@@ -1696,53 +1696,46 @@ private bool parseCommandLine(const ref Strings arguments, const size_t argc, re
                         if (num == uint.max)
                             goto Lerror;
 
+                        string generateTransitionsNumbers()
+                        {
+                            import dmd.cli : Usage;
+                            string buf;
+                            foreach (t; Usage.transitions)
+                            {
+                                if (t.bugzillaNumber !is null)
+                                    buf ~= `case `~t.bugzillaNumber~`: params.`~t.paramName~` = true;break;`;
+                            }
+                            return buf;
+                        }
+
                         // Bugzilla issue number
                         switch (num)
                         {
-                        case 3449:
-                            params.vfield = true;
-                            break;
-                        case 10378:
-                            params.bug10378 = true;
-                            break;
-                        case 14488:
-                            params.vcomplex = true;
-                            break;
-                        case 16997:
-                            params.fix16997 = true;
-                            break;
+                        mixin(generateTransitionsNumbers());
                         default:
                             goto Lerror;
                         }
                     }
                     else if (Identifier.isValidIdentifier(p + 12))
                     {
+                        string generateTransitionsText()
+                        {
+                            import dmd.cli : Usage;
+                            string buf = `case "all":`;
+                            foreach (t; Usage.transitions)
+                                buf ~= `params.`~t.paramName~` = true;`;
+                            buf ~= "break;";
+
+                            foreach (t; Usage.transitions)
+                            {
+                                buf ~= `case "`~t.name~`": params.`~t.paramName~` = true;break;`;
+                            }
+                            return buf;
+                        }
                         const ident = p + 12;
                         switch (ident[0 .. strlen(ident)])
                         {
-                        case "all":
-                            params.vtls = true;
-                            params.vfield = true;
-                            params.vcomplex = true;
-                            break;
-                        case "checkimports":
-                            params.check10378 = true;
-                            break;
-                        case "complex":
-                            params.vcomplex = true;
-                            break;
-                        case "field":
-                            params.vfield = true;
-                            break;
-                        case "import":
-                            params.bug10378 = true;
-                            break;
-                        case "intpromote":
-                            params.fix16997 = true;
-                            break;
-                        case "tls":
-                            params.vtls = true;
-                            break;
+                        mixin(generateTransitionsText());
                         default:
                             goto Lerror;
                         }

